cicecore/cicedynB/analysis/ice_history.F90

```diff
@@ -1862,11 +1863,10 @@ subroutine accum_hist (dt)
            avgct(ns) = c1
          else                      ! write averages over time histfreq
            avgct(ns) = avgct(ns) + c1
-!           if (avgct(ns) == c1) time_beg(ns) = (time-dt)/int(secday)
-           if (avgct(ns) == c1) then
-              time_beg(ns) = (timesecs-dt)/int(secday)
-              time_beg(ns) = real(time_beg(ns),kind=real_kind)
-           endif
+         endif
+         if (avgct(ns) == c1) then
```

Comment:
So this just initializes timedbl and time_beg at the beginning of the averaging interval?

Reply:
correct.  and timedbl is just a temporary local to make going from double to single a little cleaner.  time_beg is now initialized whenever hist_avg is on because we should assume it might be written to the history file when hist_avg is on, even for histfreq='1' and histfreq_n=1.  This moves away from treating histfreq='1' and histfreq_n=1 as special cases which allows for averaging across timesteps (histfreq='1') and allows a user to cleanly set hist_avg to true even if histfreq_n=1.